crates/compiler/builtins/roc/Crypt.roc

```diff
@@ -0,0 +1,123 @@
+module [
+    emptySha256,
+    sha256AddBytes,
+    sha256Digest,
+    hashSha256,
+    digest256ToBytes,
+    Sha256,
+    Digest256,
+]
+
+import Bool exposing [Eq]
+import List
+import Num exposing [U8, U64, U128]
+import Result
+import Str
+
+## Represents, as an opaque type, the state of a SHA256 cryptographic hashing function, after some (or no) data have been added to the hash.
```

Comment:
nitpick: it being an opaque type is implied, I don't think we need to explicitly mention that here.